test/integration/remote-config.spec.ts

```diff
@@ -287,4 +290,68 @@ describe('admin.remoteConfig', () => {
       expect(newTemplate.parameterGroups).to.deep.equal(VALID_PARAMETER_GROUPS);
     });
   });
+
+  describe('getServerTemplate', () => {
+    it('should return the most recently published template', () => {
+      // TODO: use global mock data once there's a REST API for server templates.
+      // Until then, copy/paste this into test server.
+      const VALID_TEMPLATE = {
+        'conditions': [
+          {
+            'name': 'ios',
+            'expression': "device.os == 'ios'"
+          },
+          {
+            'name': 'android',
+            'expression': "device.os == 'android'"
+          }
+        ],
+        'etag': 'etag-123-456',
+        'parameters': {
+          'holiday_promo_enabled': {
+            'defaultValue': {
+              'useInAppDefault': true
+            },
+            'description': 'promo indicator',
+            'valueType': 'STRING'
+          },
+          'welcome_message': {
+            'conditionalValues': {
+              'android': {
+                'value': 'welcome android text'
+              },
+              'ios': {
+                'value': 'welcome ios text'
+              }
+            },
+            'defaultValue': {
+              'value': 'welcome text 1706578886883'
+            },
+            'valueType': 'STRING'
+          }
+        },
+        'version': {
+          'description': 'asd'
+        }
+      };
+
+      const rc = getRemoteConfig();
+
+      // Set FIREBASE_REMOTE_CONFIG_URL_BASE env var to point this at a local server,
+      // eg `FIREBASE_REMOTE_CONFIG_URL_BASE=http://localhost:3000 npm run integration`
+      // Note this applies to all RC endpoints, so filter to just this test if using a
+      // server that doesn't implement all endpoints.
+      return rc.getServerTemplate()
+        .then((template: RemoteConfigServerTemplate) => {
+          // Demonstrates explicit template data type usage.
+          const templateData: RemoteConfigServerTemplateData = template.cache;
+          expect(templateData).to.deep.equal(VALID_TEMPLATE);
+          // Demonstrates config type usage.
+          const config: RemoteConfigServerConfig = template.evaluate();
+          expect(config).to.deep.equal({
+            welcome_message: 'welcome text 1706578886883'
```

Comment:
Is there any significance to this number? If we're validating template retrieval and config, should this be a welcome message response like the following? 

"Welcome to server-side Remote Config! Your template has been successfully retrieved and validated."

Reply:
Good question. The number originally came from test data up on line 42. That line puts a timestamp in the parameter value. I copied here since there's no REST API the tests can use to create a server template.

Taking a step back, I'm wondering about the value of this test. It helped us iterate on the functionality, but we had to run a local server for it to work, unlike the other tests. I'll remove it for now.